app/services/CartService.java

```diff
@@ -0,0 +1,25 @@
+package services;
+
+import io.sphere.sdk.carts.Cart;
+import io.sphere.sdk.products.ProductProjection;
+import io.sphere.sdk.products.ProductVariant;
+import pactas.models.PactasContract;
+import pactas.models.PactasCustomer;
+import play.mvc.Http;
+
+import java.util.Optional;
+
+public interface CartService {
+
+    Cart getOrCreateCart(Http.Session session);
+
+    Cart clearCart(final Cart cart);
```

Comment:
For example here is documentation necessary, it is not clear what is cleared in the cart.